core/state/snapshot/journal.go

```diff
@@ -212,9 +208,6 @@ func (dl *diskLayer) Journal(buffer *bytes.Buffer) (common.Hash, error) {
 	if dl.stale {
 		return common.Hash{}, ErrSnapshotStale
 	}
-	// Ensure the generator stats is written even if none was ran this cycle
```

Comment:
It's unnecessary to journal the progress, reasons:

If the background generation was active, the latest progress must be persisted before terminating it
If the background generation was not active, no progress has been made, no need to persist marker